app/views/users/list.html.erb

```diff
@@ -38,9 +38,12 @@
               <div class="card-info">
                 <a href="/profile/<%= user.username %>"><%= user.username %></a>
                   <div class="sub-info">
-                    <div>Last Activity : <%= t = user.revisions.order(timestamp: :desc).first.try(:created_at);time_ago_in_words(t) if (t) %></div>
-                    <div>History : <%= user.nodes.count %> <%= t('users.list.notes_and_edits') %></div>
-                    <div> Joined : <%= distance_of_time_in_words(user.created_at, Time.current, { include_seconds: false, scope: 'datetime.time_ago_in_words' }) %></div>
+                  <span id="tags">
+                  <% Tag.sort_according_to_followers(user.tags,"desc") %>
+                  <% user.tags.limit(3).each do |tag| %>
```

Comment:
Should we store `Tag.sort_according_to_followers` in a var like: 

```ruby
tags = Tag.sort_according_to_followers
```
then use
```ruby
<% tags[0..2].each do |tag| %>
```

that way your sorting will persist, and also we're not running a new query?

Reply:
yes Now, the sorting is persisting.
I've updated it.
Thanks !!!